css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-details {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-details-card {
+			align-self: flex-start;
+			backdrop-filter: blur(20px);
```

Comment:
I just realized that we need a fallback here for Firefox and Safari.  Will have a small add here in a moment.